### PR TITLE
Add support for generating a template argument file from an integration.

### PR DESCRIPTION
**Pull request by mschaef-da**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,6 @@ Available variables include the following:
 | Variable | Default | Purpose |
 |----------|---------|---------|
 | `DABL_HEALTH_PORT` | 8089 | Port for Health/Status HTTP endpoint |
-| `DABL_INTEGRATION_INTEGRATION_ID` | | Integration ID (arbitrary for testing, but required) |
```
Comment (mschaef-da) [inline README.md:76]: Turns out we didn't need this at all within an integration at runtime.

Comment (da-tanabe) [reply README.md:76]: nice

File: daml_dit_api/main/main.py [hunk 2]
```diff
@@ -50,28 +68,29 @@ def create_network(url: str) -> 'Network':
...
+    """
+    Run the dazl networks, and make sure that fatal dazl errors terminate the application.
+    """
+    LOG.info('Starting dazl network(s)...')
+    # noinspection PyBroadException
+    try:
+        futures = [ensure_future(n.aio_run()) for n in networks]
+        await gather(*futures)
+    except:  # noqa
+        FAIL('The main dazl coroutine died with an exception')
```
Comment (da-tanabe) [inline daml_dit_api/main/main.py]: It's probably not completely related to this PR, but if there is only ever one `Network` being connected to, this whole thing should probably just be simplified to `await network.aio_run()` instead.

This originally surfaced in `dabl-services` when we used to talk to both child ledgers and the admin ledger at the same time, and I'm not sure there's really a need for it any more in any of our applications.

**Review by da-tanabe — APPROVED** [review]

File: daml_dit_api/main/main.py (hunk 2 above)
Comment (mschaef-da) [reply daml_dit_api/main/main.py]: Thanks... I'd actually meant to include a fix for that in this.

Comment (mschaef-da) [reply daml_dit_api/main/main.py]: Fixed.

**Pull request merged**